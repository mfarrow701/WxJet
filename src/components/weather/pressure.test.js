import React from 'react';
import {shallow} from 'enzyme';
import Pressure from './pressure';

describe('Pressure component', () => {
    let component;

    const renderWrapper = () => {
        component = shallow(<Pressure/>);
    };

    beforeEach(() => {
        renderWrapper();
    });

    it('should render the pressure component after load', () => {
        expect(component).toMatchSnapshot();
        expect(component.find('.Pressure-Type').text()).toEqual('QNH');
        expect(component.find('.Pressure-Value').text()).toEqual('1024');
    });
});
