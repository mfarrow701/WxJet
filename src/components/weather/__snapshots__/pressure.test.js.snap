// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pressure component should render the pressure component after load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Pressure />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h1
          className="Pressure-Type"
>
          QNH
</h1>,
        <h3
          className="Pressure-Value"
>
          1024
</h3>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "QNH",
          "className": "Pressure-Type",
        },
        "ref": null,
        "rendered": "QNH",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "1024",
          "className": "Pressure-Value",
        },
        "ref": null,
        "rendered": "1024",
        "type": "h3",
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1
            className="Pressure-Type"
>
            QNH
</h1>,
          <h3
            className="Pressure-Value"
>
            1024
</h3>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "QNH",
            "className": "Pressure-Type",
          },
          "ref": null,
          "rendered": "QNH",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "1024",
            "className": "Pressure-Value",
          },
          "ref": null,
          "rendered": "1024",
          "type": "h3",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
